src/generators/cfml.ts

```diff
@@ -0,0 +1,114 @@
+import * as util from "../util.js";
+import type { Request } from "../util.js";
+
+import jsesc from "jsesc";
+
+const quote = (str: string): string => {
+  return jsesc(str, { quotes: "single" }).replace(/"/g, '""');
+};
+
+export const _toCFML = (request: Request): string => {
+  let cfmlCode = "";
+
+  cfmlCode += "httpService = new http();\n";
+  cfmlCode += 'httpService.setUrl("' + quote(request.url as string) + '");\n';
+  cfmlCode += 'httpService.setMethod("' + quote(request.method) + '");\n';
+
+  if (request.cookies) {
+    for (const [headerName, headerValue] of request.cookies) {
+      cfmlCode +=
+        'httpService.addParam(type="cookie", name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue) +
+        '");\n';
+    }
+    util.deleteHeader(request, "Cookie");
+  }
+
+  if (request.headers && request.headers.length) {
+    for (const [headerName, headerValue] of request.headers) {
+      cfmlCode +=
+        'httpService.addParam(type="header", name="' +
+        quote(headerName) +
+        '", value="' +
+        quote(headerValue as string) +
+        '");\n';
+    }
+  }
+
+  if (request.timeout) {
+    cfmlCode +=
+      'httpService.setTimeout("' + (parseInt(request.timeout) || 0) + '");\n';
```

Comment:
`httpService.setTimeout` takes a string?

Reply:
in CFML it will interpret it as a number since it is a dynamically typed language, but it will accept both, i can remove the quotes since it will always be a number from curl

Reply:
don't go away, I'm updating it right now, give me 2 minutes

Reply:
done